src/processes.c

```diff
@@ -2160,33 +2089,33 @@ static int ps_read (void)
 		pse.id       = pid;
 		pse.age      = 0;
 
-		pse.num_proc   = ps.num_proc;
-		pse.num_lwp    = ps.num_lwp;
-		pse.vmem_size  = ps.vmem_size;
-		pse.vmem_rss   = ps.vmem_rss;
-		pse.vmem_data  = ps.vmem_data;
-		pse.vmem_code  = ps.vmem_code;
-		pse.stack_size = ps.stack_size;
+		pse.data.num_proc   = ps.data.num_proc;
```

Comment:
Don't you need to change all of the other occurrences of these fields? I mean, in the other `#elif` sections?